config-store-yaml/src/main/java/io/stargate/config/store/yaml/ConfigStoreActivator.java

```diff
@@ -79,7 +79,7 @@ private synchronized void startConfigStore(Metrics metrics) {
     started = true;
 
     Hashtable<String, String> props = new Hashtable<>();
-    props.put("Identifier", CONFIG_STORE_YAML_IDENTIFIER);
+    props.put("ConfigStoreIdentifier", CONFIG_STORE_YAML_IDENTIFIER);
```

Comment:
Just wondering: What is the reason for this change?

Reply:
I see that we have this pattern for authIdentifier (having unique selector name) 
It maybe easier to find usages of it instead of using Identifier name everywhere.